<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleWindow</class>
 <widget class="QWidget" name="ConsoleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MoonEd Console</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
